pkg/client/client.go

```diff
@@ -45,9 +43,6 @@ func isDebugMode() error {
 	if debugEnv == "on" {
 		return nil
 	}
-	if viper.GetBool("Debug") {
```

Comment:
do we use this flag in our testing at all?

Reply:
we shouldnt test insecure mode anymore